src/components/AllReviews.jsx

```diff
@@ -0,0 +1,32 @@
+import { useEffect } from "react";
+import { useState } from "react";
+import ReviewCard from "./ReviewCard"
+
+export default function AllReviews() {
+    const[reviews, setReviews] = useState([]);
+
+
+    useEffect(()=>{
+        fetch("https://bencgames.herokuapp.com/api/reviews")
+        .then((res)=>{
+            return res.json()
+        }).then ((response)=>{
+            console.log(response)
```

Comment:
watch out leaving these console.logs in your production code, they can be a pain to hunt out later :)